src/contrib/talloc.rst

```diff
@@ -248,46 +248,94 @@ running process:
     FILE=/tmp/talloc.$PROCESS
     sudo gdb -quiet -batch -p $PROCESS \
         -ex "set \$file = (FILE*)fopen(\"$FILE\", \"w+\")" \
-        -ex 'call talloc_enable_null_tracking()' \
-        -ex 'call talloc_report_full(0, $file)' \
+        -ex 'call (void) talloc_report_full(autofree_ctx, $file)' \
+        -ex 'detach' \
+        -ex 'quit' &> /dev/null
+
+To allow ``gdb`` to handle the ``FILE`` type correctly and the ``autofree_ctx``
+symbol debug information should be installed at least for ``glibc`` and SSSD.
+On Fedora you can call
```

Comment:
Use code-tabs instead of saying "On Fedora", e.g. https://github.com/SSSD/sssd.io/blob/master/src/contrib/building-sssd.rst?plain=1#L33